keyboards/cue2keys/cue2keys.c

```diff
@@ -0,0 +1,406 @@
+// Copyright 2025 esplo
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include QMK_KEYBOARD_H
+#include <stdint.h>
+#include <stdbool.h>
+#include "matrix.h"
+#include "atomic_util.h"
+#include "i2clib.h"
+#include "i2c_master.h"
+
+extern DeviceList deviceList[MAX_MCP_NUM];
+extern uint16_t   nDevices;
+
+#ifdef OLED_ENABLE
+DisplayMode display_mode = DisplayMode_Layer;
+#endif
+
+static pin_t direct_pins[MATRIX_ROWS][MATRIX_COLS] = DIRECT_PINS;
+
+enum my_keycodes {
+    // Next OLED Page
+    NEXT_OLED_PAGE = QK_KB_0,
+    // Trackball angle
+    ROT_R30_1,
+    ROT_L30_1,
+    ROT_R30_2,
+    ROT_L30_2,
+    ROT_R2_1,
+    ROT_L2_1,
+    ROT_R2_2,
+    ROT_L2_2,
+    // mouse key latency
+    MOUSE_LAYER_MS_ADD_100MS,
+    MOUSE_LAYER_MS_MINUS_100MS,
+    // x0.5, x1 (default), x2, x4
+    POINTER_SPEED_MAG_CHANGE,
+};
+
+void keyboard_post_init_kb(void) {
```

Comment:
```suggestion

kb_config_t kb_config = {0};

void keyboard_post_init_kb(void) {
```

Reply:
Thank you, I added the suggested initialization.